
时间：2017-06-29 09:11:26,444 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：47818 

时间：2017-06-29 09:12:16,525 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：218 

时间：2017-06-29 09:12:34,685 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：18378 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 09:12:34,685 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：18378 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 09:13:31,963 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：6616 

时间：2017-06-29 09:14:07,042 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：41695 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 09:14:07,042 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：41695 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 09:14:35,842 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"验证码不正确 ","code":"0x0007"} 
时间：70494 

时间：2017-06-29 09:15:07,223 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：101876 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 09:15:07,243 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：101896 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 09:17:46,419 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：355 

时间：2017-06-29 09:20:50,843 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：184778 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 09:20:50,853 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：184788 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 09:27:07,908 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：225 

时间：2017-06-29 09:30:32,424 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：204740 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 09:30:32,429 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：204745 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 09:53:55,111 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：1607427 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 09:53:55,122 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：1607438 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 09:58:55,190 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：1907507 

时间：2017-06-29 11:24:30,709 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：7043025 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 11:24:30,709 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：7043025 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 11:33:59,150 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：7611466 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 11:33:59,166 
级别：FATAL 
出错类：X.Web.XApp property: [(null)] 
出错位置：X.Core.Plugin.Loger.Fatal(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:68) - 
错误：致命错误 
时间：7611482 
System.InvalidOperationException: 可为空的对象必须具有一个值。
   在 System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   在 System.Nullable`1.get_Value()
   在 X.App.Apis.mgr.user.list.<>c.<Execute>b__12_0(x_user u) 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   在 System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   在 System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   在 X.App.Apis.mgr.user.list.Execute() 位置 F:\HT\xue\src\Web\Xue\X.App\Apis\mgr\user\list.cs:行号 32
   在 X.Web.Apis.Api.GetResponse() 位置 F:\HT\xue\src\Web\X.Web\Apis\Api.cs:行号 31
   在 X.Web.XApp.ProcessRequest(HttpContext c) 位置 F:\HT\xue\src\Web\X.Web\XApp.cs:行号 31

时间：2017-06-29 13:05:44,867 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：13117183 

时间：2017-06-29 13:08:32,996 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：13285312 

时间：2017-06-29 14:50:13,533 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"缺少参数 ","code":"0x0003"} 
时间：19385849 

时间：2017-06-29 17:27:37,832 
级别：ERROR 
出错类：Error property: [(null)] 
出错位置：X.Core.Plugin.Loger.Error(F:\HT\xue\src\Xc\Core\Plugin\Loger.cs:46) - 
错误：{"issucc":false,"msg":"用户登陆超时 ","code":"0x0005"} 
时间：28830148 
